azure-pipelines.yml

```diff
@@ -9,7 +13,7 @@ stages:
 - stage: Build
   jobs:
   - job: Windows
-    timeoutInMinutes: 90
+    timeoutInMinutes: 120
```

Comment:
this seems fine, is there anything that can be done to improve build times?

Reply:
I am unsure, but created an issue:
https://github.com/microsoft/DirectXShaderCompiler/issues/7090